recipe/migrations/bullet_cpp325.patch

```diff
@@ -0,0 +1,8 @@
+migrator_ts: 1697202337
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+
+bullet_cpp:
+  - '3.25'
```

Comment:
The `run_exports` is `x.x.x` but the pin is only `x.x`. One of both needs to be adjusted.

Reply:
Yes, I also thought about this. The situation is a bit intricate. In general bullet does not offer any kind of ABI guarantees, so any version could break the ABI. Furthermore, the version scheme is not consistent: there have been major.minor releases (like the current one, `3.25`), and also major.minor.patch releases (see https://github.com/bulletphysics/bullet3/tags?after=3.08). This   major.minor.patch is the reason why the run exports is x.x.x, however the current release is 3.25 (not 3.25.0), so I thought it made more sense to have 3.25 in the pinning. Furthermore, I just noticed that the run_export x.x.x gets rendered as `"bullet-cpp >=3.25,<3.26.0a0"` in downstream packages, so I think probably in this specific case pinning to 3.25 make more sense? 

Anyhow, thinking a bit more about this I think it make more sense to just change the run_exports to x.x . If in the future the is an ABI breakage in x.x.x releases, we can deal with it via repodata patches.

Reply:
Done in https://github.com/conda-forge/bullet-feedstock/pull/62, I will ping on merge.